python/pyarrow/tests/conftest.py

```diff
@@ -184,9 +184,16 @@ def gcs_server():
     args = [sys.executable, '-m', 'testbench', '--port', str(port)]
     proc = None
     try:
+        # check first if testbench module is available
+        import testbench  # noqa:F401
```

Comment:
Not doing this adds 30+ min to the CI runs, because (on conda-forge CI, with all tests), because each test using `GcsFileSystem` runs into a lengthy timeout game during `fs.create_dir(bucket)`